### PR TITLE
Stepangranat/tg 450 refactor monorepo build system

### PR DESCRIPTION
**Pull request by stepan662**

### CODE REVIEW

**Review by JanCizmar — CHANGES_REQUESTED** [review]
Good job! Just a few things.

The angular building doesn't work for me, since @tolgee/ngx and angular-testapp are building at the same time. It fails, because anular-testapp couldn't find @tolgee/ngx module, probably because the angular builder removes the dist dir and writes to it after it's built in memory.

Also, any of the build commands hangs after start for some time ca 30sec.

File: scripts/workspacesRemove.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -x
+
+# remove "workspaces" section from package.json, so npm installs only root packages
+cp package.json package.json.original
+cat package.json.original | sed -e '/"workspaces":/,+6d' > package.json
```
[the remark this answers is not in the document]
Comment (stepan662) [reply scripts/workspacesRemove.sh:6]: Yep, that's true, I'll make it in javascpript.

File: e2e/scripts/startCypress.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (stepan662) [reply e2e/scripts/startCypress.sh:1]: 👍

File: packages/ngx/package.json [hunk 3]
```diff
@@ -14,7 +14,8 @@
     "ng": "ng",
     "develop": "ng build --watch",
     "build": "ng build",
-    "buildProd": "ng build --prod",
+    "clean": "rm -rf build build-e2e dist dist-e2e lib .angular",
+    "version-change-rebuild": "ng build --prod",
```
[the remark this answers is not in the document]
Comment (stepan662) [reply packages/ngx/package.json:18]: Hmm, can't we just build it with --prod right away? I'll try it on monday ...

File: e2e/scripts/startCypress.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (stepan662) [reply e2e/scripts/startCypress.sh:1]: I started with simple shell scripts, but then it got more complex 😄

**Pull request merged**